cranelift/codegen/src/write.rs

```diff
@@ -234,7 +234,10 @@ pub fn write_block_header(
     };
 
     // The `indent` is the instruction indentation. block headers are 4 spaces out from that.
-    write!(w, "{1:0$}{2}", indent - 4, "", block)?;
+    match func.dfg.block_name(block) {
+        Some(block_name) => write!(w, "{1:0$}{2}", indent - 4, "", block_name)?,
+        None => write!(w, "{1:0$}{2}", indent - 4, "", block)?,
+    }
```

Comment:
The block names should probably also be printed in for example `jump block1`. Also would be nice to support them in the parser too.

Reply:
oh, right. i completely forgot that...